app/components/slider/slider.css

```diff
@@ -0,0 +1,38 @@
+.horizontal-slider {
+  padding: 8px 16px 24px 16px;
+}
+
+.slider-track {
+  height: 2px;
+  background-color: gray;
+}
+
+.slider-thumb {
+  width: 32px;
+}
+
+.slider-thumb-circle {
+  margin-top: -5px;
+  width: 10px;
+  height: 10px;
+  border: 1px solid darkslategray;
+  border-radius: 10px;
+  background: darkslategray;
+}
+
+.slider-thumb-value {
+  font-size: 12px;
+  text-align: center;
+}
+
+.slider-thumb-1 .slider-thumb-circle {
+  margin-left: 20px;
+}
+
+.slider-thumb-0 .slider-thumb-value {
+  margin-left: -16px;
+}
+
+.slider-thumb-1 .slider-thumb-value {
+  margin-right: -16px;
+}
```

Comment:
To avoid negative margins here, maybe you could make the `.slider-thumb` parent (which contains the circle and the value) a column-directed flexbox with items centered vertically, like

```css
.slider-thumb {
  display: flex;
  flex-direction: column;
  align-items: center;
}
```

If you still want to nudge the slider-thumb-value text a bit, you could apply a non-negative padding-right rather than a negative margin-left.

Reply:
Done, but there's still one negative margin to get the circle onto the slider, let me know if you can see a way to get rid of that one.

Reply:
Looks like you got rid of it, nice :+1: